### PR TITLE
MON-3127: add nodeExporter.collectors.ksmd settings.

### PR DESCRIPTION
**Pull request by raptorsun**
* [x] I added CHANGELOG entry for this change.
* [ ] No user facing changes, so no entry in CHANGELOG was needed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Note: This CHANGELOG is only for the monitoring team to track all monitoring related changes. Please see OpenShift release notes for official changes.
 
+## 4.14
+
+- [#1931](https://github.com/openshift/cluster-monitoring-operator/pull/1931) add nodeExporter.collectors.ksmd settings.
```
Comment (simonpasquier) [inline CHANGELOG.md:5]: We agreed to stop using the CHANGELOG for everything that is already tracked in an epic.

File: pkg/manifests/types.go [hunk 2]
```diff
@@ -297,6 +297,9 @@ type NodeExporterCollectorConfig struct {
 	// Defines the configuration of the `buddyinfo` collector, which collects statistics about memory fragmentation from the `node_buddyinfo_blocks` metric. This metric collects data from `/proc/buddyinfo`.
 	// Disabled by default.
 	BuddyInfo NodeExporterCollectorBuddyInfoConfig `json:"buddyinfo,omitempty"`
+	// Defines the configuration of the `ksmd` collector, which collects statistics about kernel same-page merger daemon.
```
Comment (simonpasquier) [inline pkg/manifests/types.go:300]:
```suggestion
	// Defines the configuration of the `ksmd` collector, which collects statistics from the kernel same-page merger daemon.
```

**Review by simonpasquier — COMMENTED** [review]
/lgtm

**Comment by raptorsun** [comment]
Hello @bburt-rh @zhaozhanqi @Senthamilarasu-STA Could you please have a look on the PR? 
Need stamps of docs-approved, qe-approved and px-approved labels :)

**Comment by Tai-RedHat** [comment]
test PR with cluster-bot, new features implemented.
/label qe-approved

**Comment by raptorsun** [comment]
Hello  @Senthamilarasu-STA Could you please have a look at the PR and stamp it with `px-approved` label? :)

**Review by simonpasquier — APPROVED** [review]
/lgtm

**Comment by raptorsun** [comment]
fixed a typo (tab vs space in yaml) in the E2E test for node exporter. 
The test should pass now.

**Pull request merged**